src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -311,6 +311,8 @@ internal virtual int ParameterCount
             }
         }
 
+        int IMethodSymbolInternal.ParameterCount => ParameterCount;
```

Comment:
>IMethodSymbolInternal [](http://example.com/codeflow?start=12&length=21)

It looks like there is a region below dedicated to implementation of this interface.